### DISCUSSION TITLE
multi-task learning

### DISCUSSION THREAD

**Discussion by rohanvarm** [NasimAbdollahi/NodeCoder#6] [General]
Hi,
Did you see any improvement when using Multi-task learning as opposed to single task learning? In addition, do you think protein language embeddings would help with the existing framework? 

Thanks  and great work!

**Comment by NasimAbdollahi** [comment]
Hi @rohanvarm, 

This is a very good question. We trained and tested NodeCoder on a group of human proteins, for a limited number of tasks as mentioned in the paper. So I am afraid my answer cannot be generalized, and more comprehensive training and testing experiments for various tasks are required to find out about the behaviour of trained models for single-tasks vs multi-task learning. What I can share with you is that, for the conducted experiments, we did not observe improvements when performing multitask learning. 

Many thanks for your feedback!
